launchable/commands/split_subset.py

```diff
@@ -44,6 +49,31 @@
     type=click.Path(),
     multiple=True,
 )
+@click.option(
+    "--split-by-groups",
+    'is_split_by_groups',
+    help="split by groups that were set by `launchable record tests --group`",
+    is_flag=True
+)
+@click.option(
+    "--split-by-groups-with-rest",
+    'is_split_by_groups_with_rest',
```

Comment:
The spec was using --rest instead of adding this. Is there a reason for not going with the spec?

Reply:
yes, originally set rest output **file** path to the `--rest` option. Also, needs to set something value when we use the `—rest` option. e.g) launchable-rest.txt

But in split by group should produce `{OUTPUT_DIR}/rest-${GROUP_NAME}.txt`. In my understanding Set `OUTPUT_DIR` path by `---split-by-groups-output-dir` options

So, if we keep the `—rest` option, the user needs to set a dummy path `—rest hoge` to use the rest option.

Reply:
Got it. Updated the spec with the rationale.